docs/UserGuide.md

```diff
@@ -165,12 +165,61 @@ Figure 4.1. Example of how NUSave's GUI looks like.
 
 ## 5. Tips
 ### 5.1. Syntax highlighting
+(Contributed by Song Yu)
+
+When you type an invalid command into the `command box`, your input will be highlighted.
+
+Color | Description | Examples
+------------ | ------------- | ------
+Red | Invalid input | <ul><li>An unknown command <br/> e.g. test, hello </li>  <li>Invalid command parameter(s) <br/> e.g. pt/test m/hello</li></ul>
+Grey |  The default font color, or when the command is valid |
 
 ## 6. Commands
 ### 6.1. Navigation
 #### 6.1.1. Opening a budget: `open`
+(Contributed by Song Yu)
+
+You can use this command to **open a budget.**
+
+By opening a budget, the application will display the list of expenditures belonging to that budget.
+
+Once a budget is opened, the commands that are applicable to adding, deleting or editing an expenditure will now be 
+recognised as a command.
 
+Format: `open INDEX`
+
+ Parameters | Requirement | Comments
+ ------------ | ------------- | ------
+Index | ❗Required | Index of the budget
+
+✏️ Example:
+
+`open 1`
+
+This will open the budget with the index "1", based on the index as shown beside the name of the budget.
+
+![Example of open budget command](images/CommandScreenShots/6_1_1_openBudget.png)
+![Example of open budget command](images/CommandScreenShots/6_1_1_openBudget2.png)
 #### 6.1.2. Closing a budget: `close`
+(Contributed by Song Yu)
+
+You can use this command to **close a budget.**
+
+By closing a budget, the application will display all budgets in NUSave.
```

Comment:
I'm not sure if it might be a good idea to mention that the application switches from displaying expenditures to displaying budgets instead.